buildscripts/kokoro/android-interop.sh

```diff
@@ -21,7 +21,7 @@ echo y | ${ANDROID_HOME}/tools/bin/sdkmanager "build-tools;28.0.3"
 # Proto deps
 buildscripts/make_dependencies.sh
 
-GRADLE_FLAGS="-Pandroid.useAndroidX=true -Pandroid.enableJetifier=true"
```

Comment:
Removed `enableJetifier` flag as told in https://developer.android.com/jetpack/androidx/migrate

New manual test run: https://fusion2.corp.google.com/invocations/03d4b2e2-cb89-4465-a319-fc5ce4801afa